pkg/occlient/occlient.go

```diff
@@ -949,11 +1063,30 @@ func (c *Client) UpdateDCToSupervisor(commonObjectMeta metav1.ObjectMeta, compon
 		Ports:     foundCurrentDCContainer.Ports,
 	}
 
+	s2iScriptsProtocol, s2iScriptsURL, err := GetS2IScriptsPathFromBuilderImg(imageStreamImage)
+	if err != nil {
+		return errors.Wrap(err, "unable to bootstrap supervisord")
+	}
+
+	// Append s2i related parameters extracted above to env
+	inputEnvs := uniqueAppendOrOverwriteEnvVars(
+		foundCurrentDCContainer.Env,
+		corev1.EnvVar{
+			Name:  EnvS2IScriptsURL,
+			Value: s2iScriptsURL,
+		},
+		corev1.EnvVar{
+			Name:  EnvS2IScriptsProtocol,
+			Value: s2iScriptsProtocol,
+		},
+	)
+
 	// Generate the SupervisorD Config
-	dc := generateSupervisordDeploymentConfig(commonObjectMeta, componentImageType, commonImageMeta, foundCurrentDCContainer.Env)
+	dc := generateSupervisordDeploymentConfig(commonObjectMeta, componentImageType, commonImageMeta, inputEnvs)
 
 	// Add the appropriate bootstrap volumes for SupervisorD
 	addBootstrapVolumeCopyInitContainer(&dc, commonObjectMeta.Name)
+	//addBootstrapSupervisordInitContainer(&dc, commonObjectMeta.Name)
```

Comment:
Don't we need this? Why are we removing the bootstrap supervisord image?

Reply:
Well I am not very sure why I added this line and commented it out because it’s already there in the line below and is exactly the same...😀